README.md

```diff
@@ -39,14 +39,17 @@ If you want to add your app, feel free to open a pull request to add your app to
 - Superpower ChatGPT | [source](https://github.com/saeedezzati/superpower-chatgpt) | [Chrome](https://chrome.google.com/webstore/detail/superpower-chatgpt/amhmeenmapldpjdedekalnfifgnpfnkc) | [Firefox](https://addons.mozilla.org/en-US/firefox/addon/superpower-chatgpt/)
 
 ### Self-Hosted
+- Anse | [demo](https://anse.app) | [source](https://github.com/anse-app/anse)
 - ChatGPT UI | [source](https://github.com/patrikzudel/PatrikZeros-ChatGPT-API-UI)
 - ChatGPT Web | [source](https://github.com/Chanzhaoyu/chatgpt-web)
+- Chatpad AI | [demo](https://chatpad.ai) | [source](https://github.com/deiucanta/chatpad)
+- GPT4All | [source](https://github.com/nomic-ai/nomic)
```

Comment:
This is not open source, the GitHub link is just to their dataset. The website doesn't have any info other than to download a binary or contact sales https://www.nomic.ai/gpt4all

Reply:
Can move to the "Not Open Source" section